PACKAGING.md

```diff
@@ -662,3 +664,32 @@ You can even specify the path to another folder, if for example you've got anoth
 fpm package in the same repository. Like this: `helloff = { path = "helloff" }`.
 Note that you should *not* specify paths outside of your repository, or things
 won't work for your users.
+
+### Custom Build Scripts
+
+If there is something special about your library that makes fpm unable to build
+it, you can provide your own build script. fpm will then simply call your
+build script to build the library.
+
+To specify a build script to be used, put it in the library section of your
+`fpm.toml` file, like:
+
+```toml
+[library]
+source-dir="src"
+build-script="my_build_script"
+```
+
+fpm will set the following environment variables to specify some parameters to
+the build script.
+
+* `FC` - The Fortran compiler to be used
+* `FFLAGS` - The flags that should be passed to the Fortran compiler
+* `BUILD_DIR` - Where the compiled files should be placed
+* `INCLUDE_DIRS` - The folders where any dependencies can be found
+
+Additionally, script will be called with the name of the archive (`*.a` file)
+that should be produced as the command line argument.
```

Comment:
Is this a full path to the archive file including `BUILD_DIR` or just the filename?
Perhaps worth mentioning explicitly that this is only for scripts not Makefiles to avoid confusion.

Reply:
Yes, it is the full path. And it actually is also for Makefiles. This is the target specified, so your Makefile doesn't necessarily have to have the library as the default target.

Reply:
Ah I see, that makes a lot of sense. I would suggest also stating that explicitly in the following section about invoking `make`.